pipelinewise/fastsync/commons/tap_mongodb.py

```diff
@@ -298,6 +298,9 @@ def _export_collection(self, export_dir: str, collection_name)->str:
         if self.connection_config.get('replica_set', None) is not None:
             url += f'&replicaSet={self.connection_config["replica_set"]}'
 
+        if self.connection_config.get('ssl', None) is not None:
+            url += f'&ssl={self.connection_config["ssl"]}'
```

Comment:
`ssl` is an optional parameter and it might cause `KeyError: 'ssl'` at runtime.
Can we use `self.connection_config.get('ssl')` instead?

Reply:
The case when `ssl` is not set is ruled out by the if branch. Similarly as with the `replica_set` parameter 2 lines above, when the option is not specified at all, the `url` is not modified.

Reply:
ah apologies, I'm blind :(